core/admin/assets/content/assets/app.js

```diff
@@ -75,5 +125,12 @@ $('document').ready(function() {
         $('form :input').prop('disabled', true);
     }
 
+    if (window.isSecureContext) {
+        $("#HIBPpw").change(function(){
```

Comment:
I think the password fields are missing this HIBPpw id.
Shouldn't this just be "#pw"?

Reply:
Indeed; that's 24b2c7c0

I was playing with ":password" as a selector and then decided to override it in the template (but haven't committed the files) ... your suggested approach is better.